platypus/commands.py

```diff
@@ -209,7 +209,7 @@ def split_db(tax_fp, seqs_fp, query, output_fp, split_fp):
     interest_fp = open(join(output_fp, 'interest.fna'), 'w')
     rest_fp = open(join(output_fp, 'rest.fna'), 'w')
 
-    for full_name, seq in MinimalFastaParser(open(seqs_fp, 'U')):
+    for full_name, seq in parse_fasta(seqs_fp):
```

Comment:
Since parse_fasta is already tagged for deprecation, should we use read instead?